Coralogix/Sources/CoralogixRum.swift

```diff
@@ -17,21 +18,21 @@ public class CoralogixRum {
     internal var viewManager = ViewManager(keyChain: KeychainManager())
     internal var sessionManager = SessionManager()
     internal var sessionInstrumentation: URLSessionInstrumentation?
+    internal var cxMetricsManager = CXMetricsManager()
```

Comment:
dont think we should add cx prefix, its internal so lets keep it like other vars `metricsManager`